lib/util.js

```diff
@@ -233,6 +233,9 @@ exports.getVideoID = function(link) {
   if (!id) {
     throw new Error('No video id found: ' + link);
   }
+  if (!idRegex.test(id)) {
+    throw new Error('Video id ' + id + ' does not match expected format '+idRegex.toString());
```

Comment:
Spaces between `+`

What do you think of wrapping the `id` and `idRegex` in parenthesis in the string, to make it more clear where they start and end?